@navikt/core/react/src/form-summary/FormSummary.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { HTMLAttributes, forwardRef } from "react";
+import { Box } from "../layout/box";
+import FormSummaryAnswer from "./FormSummaryAnswer";
+import FormSummaryAnswers from "./FormSummaryAnswers";
+import FormSummaryEditButton from "./FormSummaryEditButton";
+import FormSummaryHeader from "./FormSummaryHeader";
+import FormSummaryHeading from "./FormSummaryHeading";
+import FormSummaryLabel from "./FormSummaryLabel";
+import FormSummaryValue from "./FormSummaryValue";
+
+interface FormSummaryComponent
+  extends React.ForwardRefExoticComponent<
+    FormSummaryProps & React.RefAttributes<HTMLDivElement>
+  > {
+  children: React.ReactNode;
+  Header: typeof FormSummaryHeader;
+  Heading: typeof FormSummaryHeading;
+  EditButton: typeof FormSummaryEditButton;
```

Comment:
`EditButton` kan kanskje være litt obskurt når komponenten vi bruker er Link. Kanskje den bare kan hete "Edit"?

Reply:
Enig. Jeg tenkte på "EditLink", men "Edit" holder sikkert. Avhengig av hvordan søknadsdialogen er implementert, kan jeg imidlertid seg for meg at noen kanskje vil måtte utføre denne handlingen med ren JS og ikke en sidenavigering. Altså med onClick uten href. Er det da innafor at man setter `href="#"` e.l., eller skal vi åpne for at man kan ha en button her i stedet? Vi skriver jo egentlig i [dok-en vår](https://aksel.nav.no/komponenter/core/link#37bdc877d948) at man alltid bør ha med href.

Reply:
👍 Er ikke noe i veien for å ha en onClick på `a`-tag, så tror det er innafor å ikke tilby muligheten til å bytte mellom a/button.